src/app/frontend/replicasetlist/replicasetlist_controller.js

```diff
@@ -28,10 +29,24 @@ export default class ReplicaSetListController {
    */
   constructor($state, replicaSets) {
     /** @export {!Array<backendApi.ReplicaSet>} */
-    this.replicaSets = replicaSets.replicaSets;
+    this.replicaSets = replicaSets.replicaSets || [];
```

Comment:
Next thing to remember is to initialize all arrays that belong to some structure. Problem here was that `ReplicaSetList` has `ReplicaSets []ReplicaSet` array and in fact when it is not initialized using `replicaSetList.ReplicaSets = make([]ReplicaSet, 0)` then this is just null pointer.